app/src/main/java/com/kodeco/android/coordplot/VerticalSlider.kt

```diff
@@ -0,0 +1,14 @@
+package com.kodeco.android.coordplot
+
+import androidx.compose.material.Slider
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+
+@Composable
+fun createVerticalSlider(modifier: Modifier=Modifier, value: Float = 0.5f, changeValue:(Float)->Unit){
```

Comment:
Note that you could probably remove this `createVerticalSlider` composable function as it is the same as the `createHorizontalSlider` function.  The name here almost indicates that the slider itself might be vertical, which is not quite the case.